src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+            'Add copy options in menu'            : 'Ajout des choix de copie dans le menu',
+            'Show username in list view'          : 'Ajout de l’utilisateur dans la liste',
+            'Show tags in list view'              : 'Ajout des étiquettes dans la liste',
+            'Show hidden custom fields'           : 'Montrer les champs cachés',
+            'Search as i type'                    : 'Recherche pendant la frappe',
+            'Search everywhere with Enter'        : 'Recherche globale avec "Entrée"',
+            'Always show search section'          : 'Toujours montrer la recherche',
+            'Notifications'                       : 'Notifications',
+            'Send Emails for'                     : 'Envoyer un courriel pour',
+            'Security issues'                     : 'En cas de problème de sécurité',
+            'Passwords shared with me'            : 'En cas d’un partage avec moi',
+            'Show Notifications for'              : 'Montrer un notification pour',
+            'Other errors'                        : 'Erreurs diverses',
+            'Danger Zone'                         : 'Zone à risque',
+            'Reset all settings'                  : 'Réinitialiser tous les paramètres',
+            'Reset'                               : 'Réinitialiser',
+            'Delete everything'                   : 'Tout effacer',
+            'DELETE EVERYTHING'                   : 'TOUT EFFACER',
+            'Account reset requested'             : 'Réinitialisation du compte requis',
+            'View'                                : 'Vue',
+            'Default'                             : 'Standard',
+            'Advanced'                            : 'Avancés',
+            'There is nothing here'               : 'Il n’y a rien ici',
+            'Click on "+" to add something'       : 'Cliquez sur le symbole "+" pour ajouter un nouvel élément',
+            'Deleted items will appear here'      : 'Les éléments supprimés apparaîtront ici',
+            'No passwords were shared with you'   : 'Aucun mot de passe n’est partagé avec vous',
+            'You did not share any passwords'     : 'Vous n’avez partagé aucun mot de passe pour le moment',
+            'Your favorites will appear here'     : 'Vos favoris apparaîtront ici',
+            'Better check the other sections'     : 'Vous devriez verifier les autres sections',
+            'That\'s probably a good sign'        : 'C’est probablement un bon signe',
+            'Go to {href}'                        : 'Aller à {href}',
+            'Figure {count}: {title}'             : 'Images {count}: {title}',
+            'Unable to fetch page: {message}.'    : 'Impossible d’atteindre la page : {message}.',
+            'Not Found'                           : 'Non trouvé',
+            'Missing something or found an error?': 'Il vous manque une fonctionnalité ou vous rencontrée un bogue ?',
```

Comment:
I just had to change this. See [my recent commit](https://github.com/marius-wieschollek/passwords/commit/c242bd66fac3931f531bbdee6610201e8ebba8ed#diff-618b561e6a14bcc0685c25539fde4f09L289).
There is now a line at the end of each handbook page that reads `Still need help? Ask in our forum!   Found an error? Tell us!` (All individual sentences)